bfvmm/include/hve/arch/intel_x64/ept/helpers.h

```diff
@@ -123,7 +123,7 @@ void map_1g(memory_map &mem_map, gpa_t gpa, hpa_t hpa,
 /// @param mattr page table entry memory attributes to be applied to the mapping
 ///
 void map_n_contig_1g(memory_map &mem_map, gpa_t gpa, hpa_t hpa, uint64_t n,
-        memory_attr_t mattr = epte::memory_attr::wb_pt);
+                     memory_attr_t mattr = epte::memory_attr::wb_pt);
```

Comment:
I really dont like that astyle de-aligns things

Reply:
Our syntax for this is different anyways, so normally this type of issue would not happen. Newlines should be done this way:

```
void map_n_contig_1g(
    memory_map &mem_map, gpa_t gpa, hpa_t hpa, 
    uint64_t n,
    memory_attr_t mattr = epte::memory_attr::wb_pt);
```